sunpy/net/fido_factory.py

```diff
@@ -8,7 +8,7 @@
 `Fido.fetch <sunpy.net.fido_factory.UnifiedDownloaderFactory.fetch>`.
 
 """
-from collections import Sequence
```

Comment:
@nabobalis Should should backport these ones for 3.8 compatibility?

Reply:
Yes.